mods/ra/sequences/infantry.yaml

```diff
@@ -64,7 +64,8 @@ e1:
 		Start: 324
 		Length: 18
 	die6: electro
-		Length: *
+		Frames: 0, 1, 2, 3, 0, 1, 2, 3, 0, 1, 2, 3, 0, 1, 2, 3, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13
```

Comment:
IMO this loops too many times to look reasonable - I would go with `0, 1, 2, 3, 0, 1, 2, 3, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13` instead.